### DISCUSSION TITLE
How to report phishing cases hosted on vercel.

### DISCUSSION THREAD

**Discussion by Redha55** [vercel/community#7386] [Help] [answered]
### Summary

A lot of threat actors are using Vercel products to create and host phishing sites targeting different banks and organizations, how can we report them to Vercel and get them suspended effectively.?

### Steps to Reproduce

.

**Comment by pawlean** [comment, marked as answer]
Hi @Redha55!

Please email abuse@vercel.com. You will receive a response once our team has reviewed your report. If you have any questions or additional information, feel free to follow up on your open cases.

**Reply by Redha55** [reply]
Hi @pawlean,

I have recently reported some phishing pages but have not received any reply from your team.

case number: 00255351

**Reply by pawlean** [reply]
Hi @Redha55!

We'll get back to you as soon as we can. 

Thank you!

**Comment by Redha55** [comment]
Hi @pawlean 

can you please check the below case?

case number: 00260318

**Comment by javierpignata** [comment]
Hi @pawlean!

Same here. Reported in June 26 with no actions/responses from Abuse team.

Ticket ID: 00249814
Also opened a thread on https://github.com/orgs/vercel/discussions/7442

Please review. Thanks!

**Comment by amyegan** [comment]
Thanks for letting us know about a site you believe is dangerous! 

**You've already done the right thing by sending your report to [abuse@vercel.com](mailto:abuse@vercel.com). Please follow up with your open case via email if you have any questions or additional information to share.**

I, and the other members of this community, do not have access to the systems and tools needed to handle phishing reports. But your ticket is assigned to the internal team that will investigate your report.

Thank you again for reporting potentially harmful sites and for your patience while reports are investigated.
